src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -420,6 +423,7 @@ export class EthereumBlockchainService implements BlockchainService {
         } catch (err: any) {
           logger.err(err)
           const { code } = err
+          Metrics.count(METRIC_NAMES.ETH_REQUEST_ERROR_TOTAL, 1, {'error_code': code})
```

Comment:
There is a case where this "should" fail. Ex. First attempt is successful but times out before it can return success. Second attempt fails because the first attempt was successful (the error is nonce expired). Instead of throwing the error we check if the first attempt was successful. 

If this count disregards the above scenario then you can also disregard this comment